tests/context.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React from 'react';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Context from '../src/context.jsx';
+import CKEditor from '../src/ckeditor.jsx';
+import EditorMock from './_utils/editor.js';
+
+configure( { adapter: new Adapter() } );
+
+class CKEditorContextMock {
+	static create( config ) {
+		return Promise.resolve( new CKEditorContextMock( config ) );
+	}
+	static destroy() {
+		return Promise.resolve();
+	}
+}
+
+describe( 'CKEditor Context Component', () => {
+	let sandbox, wrapper;
+
+	beforeEach( () => {
+		sandbox = sinon.createSandbox();
+	} );
+
+	afterEach( () => {
+		sandbox.restore();
+
+		if ( wrapper ) {
+			wrapper.unmount();
+		}
+	} );
+
+	describe( 'initialization', () => {
+		it( 'should create an instance of the ContextWatchdog', async () => {
+			wrapper = mount( <Context context={ CKEditorContextMock } /> );
+
+			const component = wrapper.instance();
+
+			expect( component.contextWatchdog ).to.be.an( 'object' );
+		} );
```

Comment:
In fact, you don't check whether `component.contextWatchdog` is an instance of `ContextWatchdog` (`@ckeditor/ckeditor5-watchdog/src/contextwatchdog`).

Reply:
That's a good point :smile: